ecommerceproj/theretailerapp/models.py

```diff
@@ -43,3 +43,20 @@ class BasketItem(models.Model):
     basket = models.ForeignKey(Basket,on_delete=models.CASCADE)
     product = models.ForeignKey(Product,on_delete=models.CASCADE)
     quantity = models.IntegerField(default = 0)
+
+class ApprovalStatus(models.Model):
```

Comment:
please replace this class with an `Enum` class `OrderStatus`
The `status` field in `Order` model should be a `CharField`.